### PR TITLE
[irods/irods#6251] use find_package for fmt (main)

### PR DESCRIPTION
**Pull request by SwooshyCueb**
In service of irods/irods#6251

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Let's wait for one more approval.

**Review by alanking — APPROVED** [review]
\# it

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**
